.vscode/settings.json

```diff
@@ -1,11 +0,0 @@
-{
-  "eslint.validate": [
-    "javascript",
-    "javascriptreact",
-    {
-      "language": "vue",
-      "autoFix": true
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-01-08T07:17:14Z" title="Wednesday, January 8th 2020, 8:17:14 am +01:00">Jan 8, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -8,4 +8,8 @@
     }
   ],
   "editor.formatOnSave": false,
+  "eslint.autoFixOnSave": true,
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
+  }
```


This is unrelated and should be in a separate PR.

If you do so, please see the curly underlines and the hint come with them in VSCode:

![Bildschirmfoto 2020-01-08 um 08 12 24](https://user-images.githubusercontent.com/25344101/71957792-f2943c80-31ee-11ea-92bf-e557a564d140.png)

![Bildschirmfoto 2020-01-08 um 08 16 01](https://user-images.githubusercontent.com/25344101/71957875-23747180-31ef-11ea-9146-cbb7ca79f4cf.png)

That would be really nice! 🤗

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-01-10T06:15:04Z" title="Friday, January 10th 2020, 7:15:04 am +01:00">Jan 10, 2020</time>_

---


I have no idea what that is. I suddenly found this in the system.